### DISCUSSION TITLE
pt/integrations/amazon-sagemaker/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#8283] [Docs]
# pt/integrations/amazon-sagemaker/

Explora Ultralytics YOLOv8 , um modelo avançado de deteção de objectos e segmentação de imagens em tempo real para várias aplicações e plataformas de hardware.

https://docs.ultralytics.com/pt/integrations/amazon-sagemaker/

**Comment by NikoXanthidis** [comment]
Gostaria de saber se esses recursos são pagos.

**Reply by glenn-jocher** [reply]
Olá! 👋 Os recursos mencionados para implementar o YOLOv8 no Amazon SageMaker não são pagos por si só. No entanto, o uso do Amazon SageMaker e outros serviços da AWS associados à implementação e execução do modelo podem incorrer em custos conforme o uso. A AWS oferece uma camada gratuita que você pode aproveitar para experimentar alguns serviços sem custo, mas é sempre bom verificar a [página de preços do SageMaker](https://aws.amazon.com/sagemaker/pricing/) para detalhes específicos. Se precisar de ajuda para estimar os custos ou tiver outras dúvidas, estou aqui para ajudar! 🚀
